dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -156,19 +157,15 @@ USER $user
 
 # Install Azure CLI
 WORKDIR /var/$user
-RUN curl -L https://aka.ms/InstallAzureCliBundled -o azure-cli_bundle.tar.gz
-RUN tar -xvzf azure-cli_bundle.tar.gz
-RUN azure-cli_bundle_*/installer
-
-# Known bug: azure keyvault cannot work behind a proxy
-# Temporary fix: upgrade the azure-keyvault package within az cli
-# TODO: if azure-cli contains newer version azure-keyvault, remove this
-RUN ~/lib/azure-cli/bin/python -m pip install azure-keyvault==0.3.7 -U
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
 
-# Install Virtual Environment
+# Install Virtual Environments
 RUN python -m virtualenv --system-site-packages env-201811
 RUN env-201811/bin/pip install ansible==2.0.0.2
 
+RUN python3 -m venv test-reporting-env0
```

Comment:
Suggest to use a more generic name and prefix with env- like the other one. how about env-python3?

Reply:
fixed!